erpnext/hr/doctype/payroll_entry/payroll_entry.py

```diff
@@ -19,8 +19,7 @@ def onload(self):
 		# check if salary slips were manually submitted
 		entries = frappe.db.count("Salary Slip", {'payroll_entry': self.name, 'docstatus': 1}, ['name'])
 		if cint(entries) == len(self.employees) and not self.salary_slips_submitted:
-			self.db_set("salary_slips_submitted", 1)
-			self.reload()
+			self.db_set("salary_slips_submitted", 1, update_modified=False, commit=True)
```

Comment:
Don't commit implicitly inside the code. You can use `flags` instead of setting the value in a field.